srml/timestamp/src/lib.rs

```diff
@@ -176,6 +183,7 @@ decl_module! {
 			<T::OnTimestampSet as OnTimestampSet<_>>::on_timestamp_set(now);
 		}
 
+		/// checks if the timestamp was set in the current block
```

Comment:
This comment will be shown above the implementation of a trait method in the documentation. And it repeats again the code.